.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 16.19.0
+  - 18.20.0
```

Comment:
Merge the latest from `main` and use that version. Using node in travis breaks the build.